yellowbrick/contrib/scatter.py

```diff
@@ -71,6 +72,10 @@ def scatterviz(X,
     markers : iterable of strings, default: ,+o*vhd
         Matplotlib style markers for points on the scatter plot points
 
+    alpha : float, default: 1.0
```

Comment:
Thank you so much for updating the quick method with the new `alpha` parameter as well as the base visualizer! I know that there is a lot of code to update and it can be easy to miss, but if you wouldn't mind, please also update Line 85-86 and pass alpha to the `ScatterVisualizer` instantiation to ensure that alpha works as expected.

Reply:
whoops! just pushed a new commit; hopefully, that sorts it, but please double-check for me (I'm pretty new to Python!)